examples/chat/chat_main.c

```diff
@@ -256,9 +256,9 @@ static int chat_parse_args(FAR struct chat_app *priv)
 
               /* set the TTY device node */
 
-              strncpy(priv->tty,
+              strlcpy(priv->tty,
                       (FAR char *)priv->argv[i] + 2,
```

Comment:
Not related to this PR, just notice: `priv->argv[I]` is already a `char *`, so cast is not needed

Reply:
Done.